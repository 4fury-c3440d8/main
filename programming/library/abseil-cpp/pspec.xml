<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>abseil-cpp</Name>
        <Homepage>https://abseil.io/</Homepage>
        <Packager>
            <Name>PisiLinux Community</Name>
            <Email>admins@pisilinux.org</Email>
        </Packager>
        <License>Apache</License>
        <IsA>library</IsA>
        <Summary>Collection of C++ library code designed to augment the C++ standard library</Summary>
        <Description>Collection of C++ library code designed to augment the C++ standard library</Description>
        <Archive sha1sum="a66c861dc95da15fb155f8852ede8a4492b7f929" type="targz">https://github.com/abseil/abseil-cpp/archive/refs/tags/20220623.1.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
        </BuildDependencies>
        -->
        <!--
        <Patches>
            <Patch>abseil-cpp.patch</Patch>
            <Patch level="1">abseil-cpp.patch</Patch>
        </Patches>
        -->
    </Source>

    <Package>
        <Name>abseil-cpp</Name>
        <RuntimeDependencies>
            <Dependency>libgcc</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="doc">/usr/share/doc</Path>
        </Files>
    </Package>

    <Package>
        <Name>abseil-cpp-devel</Name>
        <Summary>Development files for abseil-cpp</Summary>
        <RuntimeDependencies>
            <Dependency release="current">abseil-cpp</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="data">/usr/lib/cmake</Path>
            <Path fileType="library">/usr/lib/pkgconfig</Path>
        </Files>
    </Package>

    <History>
        <Update release="3">
            <Date>2022-10-22</Date>
            <Version>20220623.1</Version>
            <Comment>Version bump.</Comment>
            <Name>Mustafa Cinasal</Name>
            <Email>muscnsl@gmail.com</Email>
        </Update>
        <Update release="2">
            <Date>2022-08-05</Date>
            <Version>20220623.0</Version>
            <Comment>Version bump.</Comment>
            <Name>Mustafa Cinasal</Name>
            <Email>muscnsl@gmail.com</Email>
        </Update>
        <Update release="1">
            <Date>2021-12-10</Date>
            <Version>20211102.0</Version>
            <Comment>First release</Comment>
            <Name>PisiLinux Community</Name>
            <Email>admins@pisilinux.org</Email>
        </Update>
    </History>
</PISI>
