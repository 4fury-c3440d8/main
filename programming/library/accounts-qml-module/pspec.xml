<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>accounts-qml-module</Name>
        <Homepage>https://launchpad.net/ubuntu/</Homepage>
        <Packager>
            <Name>Pisi Linux Admins</Name>
            <Email>admins@pisilinux.org</Email>
        </Packager>
        <License>LGPLv3</License>
        <IsA>library</IsA>
        <Summary>Expose the Online Accounts API to QML applications</Summary>
        <Description>Expose the Online Accounts API to QML applications</Description>
        <Archive sha1sum="bb99ab9922ccf849fc7009ebef9f47559f443576" type="targz">https://gitlab.com/accounts-sso/accounts-qml-module/-/archive/VERSION_0.7/accounts-qml-module-VERSION_0.7.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>signond-devel</Dependency>
            <!--Dependency>libaccounts-qt5-devel</Dependency-->
            <Dependency versionFrom="5.14.2">qt5-base-devel</Dependency>
            <Dependency versionFrom="5.14.2">qt5-assistant-devel</Dependency>
            <Dependency versionFrom="5.14.2">qt5-declarative-devel</Dependency>
        </BuildDependencies>
        <Patches>
            <Patch>Build-add-qmltypes-file-to-repository.patch</Patch>
            <Patch level="1">Fix-compilation-with-Qt-5.13.patch</Patch>
        </Patches>
    </Source>

    <Package>
        <Name>accounts-qml-module</Name>
        <RuntimeDependencies>
            <Dependency>signond</Dependency>
            <!--Dependency>libaccounts-qt5</Dependency-->
            <Dependency versionFrom="5.14.2">qt5-base</Dependency>
            <Dependency versionFrom="5.14.2">qt5-declarative</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="executable">/usr/bin</Path>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="data">/usr/share</Path>
            <Path fileType="doc">/usr/share/doc</Path>
        </Files>
    </Package>
    
    <History>
        <Update release="1">
            <Date>2020-04-08</Date>
            <Version>0.6</Version>
            <Comment>First release</Comment>
            <Name>Pisi Linux Admins</Name>
            <Email>admins@pisilinux.org</Email>
        </Update>
    </History>
</PISI>
