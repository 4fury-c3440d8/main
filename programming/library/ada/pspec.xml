<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>ada</Name>
        <Homepage>https://www.ada-url.com/</Homepage>
        <Packager>
            <Name>PisiLinux Community</Name>
            <Email>admins@pisilinux.org</Email>
        </Packager>
        <License>APACHE</License>
        <IsA>library</IsA>
        <Summary>Ada is a fast and spec-compliant URL parser written in C++.</Summary>
        <Description>Ada, C++ ile yazılmış, hızlı ve spesifikasyonlara uygun bir URL ayrıştırıcısıdır.</Description>
        <Archive sha1sum="78e92abdc3cfcd4e53f789721dfeefd52be4c151" type="targz">https://github.com/ada-url/ada/archive/refs/tags/v2.9.2.zip</Archive>
        <BuildDependencies>
            <Dependency>git</Dependency>
            <Dependency>cmake</Dependency>
            <Dependency>gtest-devel</Dependency>
        </BuildDependencies>
        <!--
        <Patches>
            <Patch>ada.patch</Patch>
            <Patch level="1">ada.patch</Patch>
        </Patches>
        -->
    </Source>

    <Package>
        <Name>ada</Name>
        <!--
        <RuntimeDependencies>
            <Dependency>Dep1</Dependency>
        </RuntimeDependencies>
        -->
        <Files>
            <Path fileType="executable">/usr/bin</Path>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="doc">/usr/share/doc</Path>
        </Files>
    </Package>

    <Package>
        <Name>ada-devel</Name>
        <Summary>Development files for ada</Summary>
        <RuntimeDependencies>
            <Dependency release="current">ada</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="data">/usr/lib/cmake</Path>
            <Path fileType="library">/usr/lib/pkgconfig</Path>
        </Files>
    </Package>

    <History>
        <Update release="2">
            <Date>2024-10-03</Date>
            <Version>2.9.2</Version>
            <Comment>Uprev. See upstream changelog.</Comment>
            <Name>Bedirhan KURT</Name>
            <Email>bedirhan.kurt@outlook.com</Email>
        </Update>
        <Update release="1">
            <Date>2024-08-02</Date>
            <Version>2.9.0</Version>
            <Comment>First release</Comment>
            <Name>PisiLinux Community</Name>
            <Email>admins@pisilinux.org</Email>
        </Update>
    </History>
</PISI>
