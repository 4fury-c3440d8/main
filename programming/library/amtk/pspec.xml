<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "https://pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>amtk</Name>
        <Homepage>https://wiki.gnome.org/Projects/Amtk</Homepage>
        <Packager>
            <Name>PisiLinux Community</Name>
            <Email>admins@pisilinux.org</Email>
        </Packager>
        <License>LGPL2</License>
        <PartOf>programming.library</PartOf>
        <IsA>app:gui</IsA>
        <Summary>Actions, Menus and Toolbars Kit for GTK+ applications</Summary>
        <Description>Actions, Menus and Toolbars Kit for GTK+ applications</Description>
        <Archive sha1sum="644eef0c877bb2597948ba1a06c73c1c09f39a60" type="tarxz">mirrors://gnome/amtk/5.3/amtk-5.3.1.tar.xz</Archive>
        <BuildDependencies>
            <Dependency>gettext-devel</Dependency>
            <Dependency>glib2-devel</Dependency>
            <Dependency>glib2-devel</Dependency>
            <Dependency>gtk-doc</Dependency>
            <Dependency>gtk3-devel</Dependency>
            <Dependency>gobject-introspection-devel</Dependency>
            <Dependency>meson</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>amtk</Name>
        <RuntimeDependencies>
            <Dependency>gtk3</Dependency>
            <Dependency>glib2</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="library">/usr</Path>
            <Path fileType="doc">/usr/share</Path>
            <Path fileType="man">/usr/share/man</Path>
         </Files>
    </Package>

    <Package>
        <Name>amtk-devel</Name>
        <RuntimeDependencies>
            <Dependency release="current">amtk</Dependency>
            <Dependency>glib2-devel</Dependency>
            <Dependency>gtk3-devel</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="library">/usr/lib/pkgconfig</Path>
            <Path fileType="header">/usr/include</Path>
         </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2021-06-02</Date>
            <Version>5.3.1</Version>
            <Comment>First release</Comment>
            <Name>Ayhan Yalçınsoy</Name>
            <Email>ayhanyalcinsoy@pisilinux.org</Email>
        </Update>
    </History>
</PISI>
