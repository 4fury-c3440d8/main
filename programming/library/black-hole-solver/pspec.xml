<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "http://www.pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>black-hole-solver</Name>
        <Homepage>https://www.shlomifish.org/open-source/projects/black-hole-solitaire-solver/</Homepage>
        <Packager>
            <Name>Pisi Linux Admins</Name>
            <Email>admin@pisilinux.org</Email>
        </Packager>
        <License>BSD</License>
        <IsA>library</IsA>
        <Summary>The Black Hole Solitaire Solver Executable</Summary>
        <Description>The Black Hole Solitaire Solver Executable</Description>
        <Archive sha1sum="92ae013bd296097095a0a5ec3918a98d3c555acf" type="tarxz">https://fc-solve.shlomifish.org/downloads/fc-solve/black-hole-solver-1.10.1.tar.xz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
            <Dependency>perl-path-tiny</Dependency>
            <Dependency>python3-devel</Dependency>
            <Dependency>rinutils-devel</Dependency>
        </BuildDependencies>
        <!--
        <Patches>
            <Patch>black-hole-solver.patch</Patch>
            <Patch level="1">black-hole-solver.patch</Patch>
        </Patches>
        -->
    </Source>

    <Package>
        <Name>black-hole-solver</Name>
        <Files>
            <Path fileType="executable">/usr/bin</Path>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="data">/usr/share</Path>
            <Path fileType="man">/usr/share/man</Path>
            <Path fileType="doc">/usr/share/doc</Path>
        </Files>
    </Package>

    <Package>
        <Name>black-hole-solver-devel</Name>
        <Summary>Development files for black-hole-solver</Summary>
        <RuntimeDependencies>
            <Dependency release="current">black-hole-solver</Dependency>
        </RuntimeDependencies>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="data">/usr/lib/cmake</Path>
            <Path fileType="library">/usr/lib/pkgconfig</Path>
        </Files>
    </Package>

    <History>
        <Update release="1">
            <Date>2021-05-10</Date>
            <Version>1.10.1</Version>
            <Comment>First release</Comment>
            <Name>Pisi Linux Admins</Name>
            <Email>admin@pisilinux.org</Email>
        </Update>
    </History>
</PISI>
