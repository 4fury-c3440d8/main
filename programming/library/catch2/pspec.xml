<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "https://pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>catch2</Name>
        <Homepage>https://github.com/catchorg/Catch2/</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safaariman@pisilinux.org</Email>
        </Packager>
        <License>BSLv1</License>
        <IsA>library</IsA>
        <Summary>C++ Automated Test Cases in Headers</Summary>
        <Description>Catch2 is a multi-paradigm test framework for C++, which also supports Objective-C. It is primarily distributed as a single header file, although certain extensions may require additional headers.</Description>
        <Archive sha1sum="00e054a9ff83928305ed790370a8ad50e411430b" type="targz">https://github.com/catchorg/Catch2/archive/refs/tags/v3.7.0.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
            <Dependency>ninja</Dependency>
            <Dependency>python3-devel</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>catch2</Name>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="library">/usr/lib</Path>
            <Path fileType="data">/usr/share</Path>
        </Files>
    </Package>

    <History>
        <Update release="3">
            <Date>2024-08-18</Date>
            <Version>3.7.0</Version>
            <Comment>Version bump.</Comment>
            <Name>fury</Name>
            <Email>uglyside@yandex.ru</Email>
        </Update>
        <Update release="2">
            <Date>2022-11-05</Date>
            <Version>2.13.9</Version>
            <Comment>Version bump.</Comment>
            <Name>Mustafa Cinasal</Name>
            <Email>muscnsl@gmail.com</Email>
        </Update>
        <Update release="1">
            <Date>2020-03-17</Date>
            <Version>2.11.1</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safaariman@pisilinux.org</Email>
        </Update>
    </History>
</PISI>
