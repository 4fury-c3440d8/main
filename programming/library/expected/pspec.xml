<?xml version="1.0" ?>
<!DOCTYPE PISI SYSTEM "https://pisilinux.org/projeler/pisi/pisi-spec.dtd">
<PISI>
    <Source>
        <Name>expected</Name>
        <Homepage>https://github.com/TartanLlama/expected</Homepage>
        <Packager>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Packager>
        <License>CC0</License>
        <IsA>library</IsA>
        <Summary>Single header implementation of std::expected with functional-style extensions.</Summary>
        <Description>std::expected is proposed as the preferred way to represent object which will either have an expected value, or an unexpected value giving information about why something failed. Unfortunately, chaining together many computations which may fail can be verbose, as error-checking code will be mixed in with the actual programming logic. This implementation provides a number of utilities to make coding with expected cleaner.</Description>
        <Archive sha1sum="ae311c9ad746d944d18e8cf873d0b1f04bff4acb" type="targz">https://github.com/TartanLlama/expected/archive/refs/tags/v1.1.0.tar.gz</Archive>
        <BuildDependencies>
            <Dependency>cmake</Dependency>
            <Dependency>ninja</Dependency>
            <Dependency>catch2</Dependency>
        </BuildDependencies>
    </Source>

    <Package>
        <Name>expected-devel</Name>
        <Files>
            <Path fileType="header">/usr/include</Path>
            <Path fileType="data">/usr/share</Path>
        </Files>
    </Package>

    <History>
        <Update release="2">
            <Date>2024-08-18</Date>
            <Version>1.1.0</Version>
            <Comment>Version bump.</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
        <Update release="1">
            <Date>2020-03-17</Date>
            <Version>1.0.0</Version>
            <Comment>First release</Comment>
            <Name>Safa Arıman</Name>
            <Email>safa@ariman.gen.tr</Email>
        </Update>
    </History>
</PISI>
